test/framework/src/main/java/org/opensearch/cluster/coordination/CoordinationStateTestCluster.java

```diff
@@ -143,8 +147,23 @@ static class ClusterNode {
                     0L
                 )
             );
+            remotePersistedState = new InMemoryPersistedState(
+                0L,
+                clusterState(
+                    0L,
+                    0L,
+                    localNode,
+                    CoordinationMetadata.VotingConfiguration.EMPTY_CONFIG,
+                    CoordinationMetadata.VotingConfiguration.EMPTY_CONFIG,
+                    0L
+                )
+            );
+            persistedStateRegistry = new PersistedStateRegistry();
+            persistedStateRegistry.addPersistedState(PersistedStateType.LOCAL, persistedState);
+            persistedStateRegistry.addPersistedState(PersistedStateType.REMOTE, remotePersistedState);
```

Comment:
we shouldnt add remoteState by default unless setting is enabled, otherwise this will mask issues where remote is disabled.

Reply:
This applies to all the tests across different classes where you are adding RemotePersistedState in registry